src/playback/qml/MuseScore/Playback/internal/MixerMuteAndSoloSection.qml

```diff
@@ -65,7 +65,7 @@ MixerPanelSection {
                 }
 
                 onToggled: {
-                    channelItem.mutedManually = !checked
+                    channelItem.mutedClicked()
```

Comment:
No problem, I'll make the implementation the same as before. Do you think it should still be called mutedManually, though? Given the fact that it can happen automatically now?